src/wp-includes/class-wp-script-modules.php

```diff
@@ -248,7 +248,7 @@ public function print_import_map() {
 		$import_map = $this->get_import_map();
 		if ( ! empty( $import_map['imports'] ) ) {
 			wp_print_inline_script_tag(
-				wp_json_encode( $import_map, JSON_HEX_TAG | JSON_HEX_AMP ),
+				wp_json_encode( $import_map, JSON_HEX_TAG | JSON_HEX_AMP | JSON_UNESCAPED_SLASHES ),
```

Comment:
I don't think there's any reason to escape `JSON_HEX_AMP` generally. Let's remove that and be consistent in the usage:

```suggestion
				wp_json_encode( $import_map, JSON_HEX_TAG | JSON_UNESCAPED_SLASHES ),
```

This was added in [[57269]](https://core.trac.wordpress.org/changeset/57269), developed in https://github.com/WordPress/wordpress-develop/pull/5818.

The only reason I'm aware of to escape `&` is for XHTML support, and that should be handled by CDATA wrappers as mentioned in https://github.com/WordPress/wordpress-develop/pull/5818#discussion_r1435559924.

Pinging @luisherranz @swissspidy from the linked conversation in case there are specific concerns about the `&` character.